modules/ROOT/pages/to-manage-categories.adoc

```diff
@@ -4,16 +4,16 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Exchange Administrators and Organization Owners can create categories to group Exchange access types.
-Categories let users organize assets into different groups to improve asset browsing and discovery.
+Use categories to organize Exchange assets into groups and improve asset browsing and discovery.
```

Comment:
Excellent rewording!
Is "and improve" more precise than "to improve"? Does the first cause the second, or are they separate things?

Reply:
The first causes the second, so I think "to improve" is more precise. Updated to use that.